inductiva/tasks/run_simulation.py

```diff
@@ -27,16 +39,58 @@ def run_simulation(
     if machine_group is not None:
         resource_pool_id = machine_group.id
 
-    task_id = methods.invoke_async_api(
+    if api_invoker is None:
+        api_invoker = methods.invoke_async_api
+
+    if not format_utils.getenv_bool("DISABLE_TASK_METADATA_LOGGING", False):
+        metadata = {
+            "storage_dir": str(storage_dir),
+            "api_method_name": api_method_name,
+            "machine_group_id": resource_pool_id,
+            **kwargs,
+        }
```

Comment:
Change the `storage_dir` to last, since it will be the least used.

```suggestion
        metadata = {
            "api_method_name": api_method_name,
            "machine_group_id": resource_pool_id,
            "storage_dir": str(storage_dir),
            **kwargs,
         }
```